test/src/main/configurations/TX/MonitorConfig.xml

```diff
@@ -1,76 +1,51 @@
-			<alarm eventCode="Sender Exception Caught" severity="CRITICAL" threshold="10" period="3600"/>
-		</monitor>
-		<monitor name="Invalid XML Message" type="FUNCTIONAL">
-			<alarm eventCode="Invalid XML: does not comply to XSD" severity="WARNING"/>
-			<alarm eventCode="Invalid XML: parser error" severity="WARNING"/>
-		</monitor>
-		<monitor name="Configuration error" type="TECHNICAL" destinations="GALM">
-			<alarm eventCode="Exception Configuring Receiver" severity="CRITICAL"/>
-		</monitor>
-		<monitor name="Message size exceeding" type="TECHNICAL">
-			<alarm eventCode="Pipe Message Size Exceeding" severity="WARNING"/>
-		</monitor>
-		<monitor name="Message in Error" type="FUNCTIONAL" destinations="GALM">
-			<alarm severity="WARNING">
-				<events>
-					<event>Receiver Moved Message to ErrorStorage</event>
-				</events>
-				<filters filterExclusive="false"/>
-			</alarm>
-			<clearing severity="WARNING">
-				<events>
-					<event>Receiver Moved Message to ErrorStorage</event>
-				</events>
-				<filters filterExclusive="false"/>
-			</clearing>
-		</monitor>
-		<monitor name="Receiver Shutdown" type="TECHNICAL" destinations="GALM">
-			<alarm severity="WARNING">
-				<events>
-					<event>Receiver Shutdown</event>
-				</events>
-				<filters filterExclusive="false"/>
-			</alarm>
-			<clearing severity="WARNING">
-				<events>
-					<event>Receiver Shutdown</event>
-				</events>
-				<filters filterExclusive="false"/>
-			</clearing>
-		</monitor>
-	</monitors>
+	<destination name="GALM" className="nl.nn.adapterframework.monitoring.GalmMonitorAdapter"/>
+	<destination name="log" className="nl.nn.adapterframework.monitoring.SenderMonitorAdapter">
+		<sender className="nl.nn.adapterframework.senders.LogSender" logCategory="monitoring"/>
+	</destination>
+	<monitor name="Internal adapter exception" type="TECHNICAL" destinations="GALM,log">
+		<trigger className="nl.nn.adapterframework.monitoring.Alarm" eventCode="Pipe Exception" severity="CRITICAL" threshold="1" period="3600">
+			<adapterfilter adapter="TransactionTimeoutHandlingNonTransacted" />
+			<adapterfilter adapter="TransactionTimeoutHandlingTransacted" />
+		</trigger>
+	</monitor>
+	<monitor name="Sender timeout" type="TECHNICAL" destinations="GALM">
+		<trigger className="nl.nn.adapterframework.monitoring.Alarm" eventCode="Sender Timeout" severity="CRITICAL" threshold="10" period="3600">
+			<adapterfilter adapter="TransactionHandlingMandatory">
+				<source>Pipe [Send transacted message to mandatory subadapter]</source>
+			</adapterfilter>
+			<adapterfilter adapter="TransactionHandlingMandatorySubAdapter">
+				<source>Pipe [Continue if this call was transacted]</source>
+			</adapterfilter>
+		</trigger>
+	</monitor>
+	<monitor name="Sender exception" type="TECHNICAL" destinations="GALM">
+		<trigger className="nl.nn.adapterframework.monitoring.Alarm" eventCode="Sender Exception Caught" severity="CRITICAL" threshold="10" period="3600"/>
```

Comment:
In plaats van 
`<trigger className="nl.nn.adapterframework.monitoring.Alarm" ... />`
verwacht ik
`<AlarmTrigger ... />`